modules/ROOT/pages/modularizing-your-configuration-files-for-team-development.adoc

```diff
@@ -4,51 +4,44 @@ include::_attributes.adoc[]
-the reality is that a gigantic XML file quickly becomes unmanageable. This is why
- it is recommended to split monolithic configurations into several files and leverage
-  Mule's capacity to load multiple configuration files at application start-up time.
- Moreover, splitting configurations into multiple fragments encourages re-use across teams.
+Though it seems convenient to keep your Mule configurations in one file,
+a very large XML file quickly becomes unmanageable. This is why
+ Mulesoft recommends that you split monolithic configurations into several files, and leverage Mule's capacity to load multiple configuration files at application start-up time.
+ Splitting configurations into multiple files encourages re-use across teams.
 
-Mule offers two options for loading several configuration files:
+Mule offers two options for loading multiple configuration files:
 
-* side-by-side: provide a list of independent configuration files to load, +
-* imported: have one configuration file import several others, which in-turn can
+* Side-by-side: Provide a list of independent configuration files to load.
+* Imported: Have one configuration file import several others, which in-turn, can
 import other files.
 
 In practice, it is common to use both approaches simultaneously.
 
-Don't forget that all the configuration files end up loaded in the same context;
-therefore you should be careful and use unique names for all your configuration
-elements. Mule will refuse to load an application whose configuration files contain
- name conflicts.
-
-How can you determine what constitutes good separation lines between configuration
- fragments? Here are a few rules of thumb:
-
-* Business domains usually form a natural border that can be used to separated
-configuration elements +
- * Keeping together elements that have similar reasons for change reduces the risk
-  of impacting unrelated aspects of your application +
- * Technical aspects, like administrative components, security or external system
- configurations, define good lines of demarcation +
- * Extracting a side-by-side connector configuration facilitates functional testing.
- Note that it is not intended to take care of environment specific connector
- configuration, which is dealt with properties files +
- * And, last but not least, re-use across teams and projects
+Because configuration files end up loaded in the same context,
+you must use unique names for the configuration elements. Mule will not load an application if its configuration files have name conflicts.
+
+Here are some suggested ways to determine where to split configuration elements into separate files and which elements to keep together:
+
+* Business domains usually form a natural border that you can use to separate
+configuration elements.
+* Keep together elements that have similar reasons for change. This reduces the risk of impacting unrelated aspects of your application.
```

Comment:
"similar reasons for change" sounds awkward

Reply:
"This" needs to modify something.

Reply:
Reworded first sentence. The "This" is clear, so I am leaving it in.